tlatools/org.lamport.tlatools/src/tla2sany/semantic/SemanticNode.java

```diff
@@ -250,7 +250,7 @@ public LinkedList<SemanticNode> get() {
 
 					@Override
 					public void preVisit(final SemanticNode node) {
-						if (location.equals(node.getLocation())) {
+						if (location.equals(node.getLocation()) || (0 == node.getChildren().length && node.getLocation().includes(location))) {
```

Comment:
`SemanticNode#getChildren` can return null > possible NPE.

Reply:
Not just hypothetical; just happened when toying with the PR.

Reply:
Note the comment about elems of getChildren potentially being `null` in `tla2sany.semantic.SemanticNode#getListOfChildren`

Reply:
https://github.com/tlaplus/tlaplus/commit/81ed3ffbbcb71ef70c3a1f6721c9a113a77f09c1